src/optimizer/stats/stats_storage.cpp

```diff
@@ -17,6 +17,7 @@
 #include "concurrency/transaction_manager_factory.h"
 #include "optimizer/stats/column_stats.h"
 #include "optimizer/stats/table_stats.h"
+#include "storage/database.h"
```

Comment:
Yes. Otherwise, it will throw an incomplete type error.  This is because the old stats framework include storage/database.h and lots of files include the old framework.